adoc/admin-security-rbac.adoc

```diff
@@ -6,34 +6,50 @@ RBAC uses the `rbac.authorization.k8s.io` API group to drive authorization decis
 
 == Authentication Flow
 
-Authentication is composed of two open source libraries:
+Authentication is composed of:
 
 * link:https://github.com/dexidp/dex[Dex] is an identity provider service (idP) that uses link:https://openid.net/connect/[OIDC] (Open ID Connect) to drive authentication for client application. It acts as portal to defer authentication to provider through connected identity providers(connectors).
-* link:https://github.com/heptiolabs/gangway[Gangway], a client application that enables authentication flow for your {productname}. User can login, authorize access and self-configure `kubectl`.
+* Client
+  . Web browser: link:https://github.com/heptiolabs/gangway[gangway], a web application that enables authentication flow for your {productname}. User can login, authorize access, download `kubeconfig` or self-configure `kubectl`.
+  . Command line: `skuba auth login`, a CLI application that enables authentication flow for your {productname}. User can login, authorize access, and get `kubeconfig`.
 
 For RBAC, administrators can use `kubectl` to create corresponding `RoleBinding` or `ClusterRoleBinding` for a user or group to limit resource access.
 
-image::oidc_flow.png[]
-// Source: suse-rbac-oidc-flow.xml (open with http://draw.io/app)
+=== Web Flow
+image::oidc_flow_web.png[]
+// Source: suse-rbac-oidc-flow-web.xml (open with http://draw.io/app)
 
 . User requests access through Gangway.
 . Gangway redirects to Dex.
 . Dex redirects to connected identity provider (connector). User login and approve access request are generated.
-. Dex continues with OIDC authentication flow on behalf of the user and creates/updates data to {kube}  CRDs.
+. Dex continues with OIDC authentication flow on behalf of the user and creates/updates data to {kube} CRDs.
 . Dex redirects the user to Gangway. This redirect includes (ID/refresh) tokens.
-. Gangway returns `kubectl` self-configure instructions to the user.
+. Gangway returns a link to download `kubeconfig` or self-configure `kubectl` instructions to the user.
 +
 image::rbac-configure-kubectl.png[]
 
 
-. User self-configures `kubectl`.
-. The `kubectl` connects to {kube} API-server.
-. {kube} CRDs validate {kube} API-server request and return response.
-. The `kubectl` connects to authorized {kube} resources through {kube} API-server.
+. User downloads `kubeconf` or self-configure `kubectl`.
+. User uses `kubectl` to connect {kube} apiserver.
+. {kube} CRDs validate {kube} apiserver request and return response.
+. The `kubectl` connects to authorized {kube} resources through {kube} apiserver.
+
+=== CLI Flow
+image::oidc_flow_cli.png[]
+// Source: suse-rbac-oidc-flow-cli.xml (open with http://draw.io/app)
+
+. User requests access through `skuba auth login` with Dex server URL, login username and password.
+. Dex receives login username and password, and do login and approve access request to the connected identity providers(connectors).
```

Comment:
Dex uses received login username and password to login and approve access request to the connected identity providers(connectors).